client/state/login/reducer.js

```diff
@@ -78,6 +82,20 @@ export const twoFactorAuthPushPoll = createReducer( { inProgress: false, success
 	[ TWO_FACTOR_AUTHENTICATION_PUSH_POLL_COMPLETED ]: state => ( { ...state, inProgress: false, success: true } ),
 } );
 
+export const socialAccount = createReducer( { creating: false }, {
+	[ SOCIAL_CREATE_ACCOUNT_REQUEST ]: () => ( { creating: true } ),
+	[ SOCIAL_CREATE_ACCOUNT_FAILURE ]: ( state, { error } ) => ( { creating: false, createError: error } ),
+	[ SOCIAL_CREATE_ACCOUNT_SUCCESS ]: ( state, { data: { username, bearerToken } } ) => ( {
+		creating: false,
+		username,
+		bearerToken
```

Comment:
I don't think that changes something from what it used to be, as this state branch is not persisted, so on reload that state is lost.